ios-base/Common/Models/Session.swift

```diff
@@ -14,6 +14,8 @@ struct Session: Codable {
   var accessToken: String?
   var expiry: Date?
   
+  var isValid: Bool { [uid, accessToken, client].allSatisfy { $0 != nil } }
```

Comment:
```suggestion
  var isValid: Bool { 
    [uid, accessToken, client].allSatisfy { $0 != nil }
  }
```

Reply:
@LetoFranco we should also check that the values are not empty strings. See the previous implementation.